scripts/system/create/edit.js

```diff
@@ -562,8 +563,7 @@ var toolBar = (function () {
             if (!properties.grab) {
                 properties.grab = {};
                 if (Menu.isOptionChecked(MENU_CREATE_ENTITIES_GRABBABLE) &&
-                    !(properties.type === "Zone" || properties.type === "Light"
-                    || properties.type === "ParticleEffect" || properties.type === "Web")) {
+                    !(properties.type === "Zone" || properties.type === "Light" || properties.type === "ParticleEffect" || properties.type === "Web")) {
```

Comment:
Line too long: should be <= 128 characters. (Coding standard 2.1.4.)

Reply:
This is not my code this is from the "screenshare" code that I had to merge because it wasn't present in my branch.
Any suggestion to shorten this ? 
I could find a way, but if you already have an idea let me know.

Reply:
Can just wrap the line onto two lines like it was originally.

Reply:
Yeah... simply put on 2 lines.